src/ragas/cli.py

```diff
@@ -457,6 +457,217 @@ def evals(
+        table.add_column("Name", style="green")
+        table.add_column("Description", style="white")
+
+        for template_id, template_info in templates.items():
+            table.add_row(
+                template_id, template_info["name"], template_info["description"]
+            )
+
+        console.print(table)
+        console.print("\n[bold]Usage:[/bold]")
+        console.print("  ragas quickstart [template_name]")
+        console.print("\n[bold]Example:[/bold]")
+        console.print("  ragas quickstart rag_eval")
+        console.print("  ragas quickstart agent_evals --output-dir ./my-project\n")
+        return
+
+    # Validate template name
+    if template not in templates:
+        error(f"Unknown template: {template}")
+        console.print(f"\nAvailable templates: {', '.join(templates.keys())}")
+        console.print("Run 'ragas quickstart' to see all available templates.")
+        raise typer.Exit(1)
+
+    template_info = templates[template]
+
+    # Find the examples directory in the installed package
+    try:
+        import ragas_examples
+
+        if ragas_examples.__file__ is None:
+            error("Could not locate ragas-examples package file.")
+            raise typer.Exit(1)
+        examples_root = Path(ragas_examples.__file__).parent
+    except ImportError:
+        error("ragas-examples package not found.")
+        console.print("\nTo install examples:")
+        console.print("  pip install ragas[examples]")
+        console.print("  # or for development:")
+        console.print("  cd /path/to/ragas && uv pip install -e ./examples")
+        raise typer.Exit(1)
+
+    source_path = examples_root / template.replace("ragas_examples/", "")
+
+    if not source_path.exists():
+        error(f"Template source not found: {source_path}")
+        raise typer.Exit(1)
+
+    # Determine output directory
+    output_path = Path(output_dir) / template
+
+    if output_path.exists():
+        warning(f"Directory already exists: {output_path}")
+        overwrite = typer.confirm("Do you want to overwrite it?", default=False)
+        if not overwrite:
+            info("Operation cancelled.")
+            raise typer.Exit(0)
+        shutil.rmtree(output_path)
+
+    # Copy the template
+    with Live(
+        Spinner(
+            "dots", text=f"Creating {template_info['name']} project...", style="green"
+        ),
+        console=console,
+    ) as live:
+        live.update(Spinner("dots", text="Copying template files...", style="green"))
+        shutil.copytree(source_path, output_path)
+        time.sleep(0.3)
+
+        live.update(
+            Spinner("dots", text="Setting up project structure...", style="green")
+        )
+
+        # Create additional directories if they don't exist
+        (output_path / "datasets").mkdir(exist_ok=True)
+        (output_path / "experiments").mkdir(exist_ok=True)
+        (output_path / "logs").mkdir(exist_ok=True)
+        time.sleep(0.2)
```

Comment:
so the structure I had in mind was the application code will be outside in the root but everything else related to evals will be inside a "evals" folder.

this is to allude to the fact that this is the pattern we are expecting